src/index.js

```diff
@@ -0,0 +1,151 @@
+const state = {
+    currentTemp: 70,
+    backgroundColor: document.getElementById('buttom--left'),
+    landscapeImg: document.getElementById('buttom--right'),
+    cityInput: document.getElementById('city-input'),
+    location: null,
+    lat: null,
+    lon: null,
+    skyCondition: document.getElementById('sky--options'),
+    skyImg: document.getElementById('top')
+}
+// changes the temp on the page
+const changesTempDisplay = () => {
+    const tempControl = document.getElementById('temperature');
+    tempControl.textContent = `${state.currentTemp}`
+}
+// CHANGES TEMP
+const increaseTemp = () => {
+    state.currentTemp++;
+    changesTempDisplay()
+};
+
+const decreaseTemp = () => {
+    state.currentTemp--;
+    changesTempDisplay()
+};
+
+// sky background change with sky option
+const skyBackground = () => {
+    console.log('skyBackground')
+    const condition = state.skyCondition.value
+    if(condition == 'sunny') {
+        console.log('sunny')
+        state.skyImg.style.backgroundImage = 'url(assets/sky/kyaw-zay-ya-8vHAfhWoqEQ-unsplash.jpg)';
+    } else if(condition == 'rainy') {
+        state.skyImg.style.backgroundImage = 'url(assets/sky/eutah-mizushima-F-t5EpfQNpk-unsplash.jpg)';
+    } else if(condition == 'snowy'){
+        state.skyImg.style.backgroundImage = 'url(assets/sky/fabrizio-conti-Mbm0WnJ5emc-unsplash.jpg)';
+    } else if(condition == 'cloudy') {
+        state.skyImg.style.backgroundImage = 'url(assets/sky/daoudi-aissa-Pe1Ol9oLc4o-unsplash.jpg)';
+    }
+
+    }
+
+// CHANGE BACKGROND COLOR  AND IMAGE WITH TEMP CHANGE
+const changeBackgrounds = () => {
+    if (state.currentTemp >= 80) {
+        state.backgroundColor.style.backgroundColor = 'red';
+        state.landscapeImg.style.backgroundImage = 'url(assets/landscape/cody-doherty-AeqlmVWtzFA-unsplash.jpg)';
+    } else if (state.currentTemp >= 70) {
+        state.backgroundColor.style.backgroundColor = 'orange';
+        state.landscapeImg.style.backgroundImage = 'url(assets/landscape/alexandr-hovhannisyan-RkOtjbPuHZw-unsplash.jpg)';
+    } else if (state.currentTemp >= 60) {
+        state.backgroundColor.style.backgroundColor = 'yellow';
+        state.landscapeImg.style.backgroundImage = 'url(assets/landscape/andrew-neel-a_K7R1kugUE-unsplash.jpg)';
+    } else if (state.currentTemp >= 50) {
+        state.backgroundColor.style.backgroundColor = 'green';
+        state.landscapeImg.style.backgroundImage = 'url(assets/landscape/joel-jasmin-forestbird-4S3iMBIappo-unsplash.jpg)';
+    } else {
+        state.backgroundColor.style.backgroundColor = 'teal';
+    }
+}
```

Comment:
Another way to format this method would be to have an object that has temp values as keys and then you could have values be a list and the list could hold background color and background image, like:

```javascript
const 80_DEGREE_IMG = 'url(assets/landscape/cody-doherty-AeqlmVWtzFA-unsplash.jpg)';
const 70_DEGREE_IMG = 'url(assets/landscape/alexandr-hovhannisyan-RkOtjbPuHZw-unsplash.jpg)';

const tempDisplayObject = {80: ['red',  80_DEGREE_IMG], 70: ['orange', 70_DEGREE_IMG]} 
```

When you have an object, you don't need to do if/else if/else if/ else which can introduce bugs and also adds complexity to the method. If you have an object, than you can iterate over the object and check the keys with state.currentTemp and then get the correct values to set backgroundColor and backgroundImage